JumpscaleCore/tools/executor/ExecutorSSH.py

```diff
@@ -21,14 +20,19 @@ def __init__(self, sshclient):
 
         self.type = "ssh"
 
-        # self.__check_base = None
-
-    # def executeRaw(self, cmd, die=True, showout=False):
-    #     return self.sshclient.execute(cmd, die=die, showout=showout)
+    def execute(
+        self, cmd, die=True, showout=False, timeout=1000, env=None, sudo=False, replace=True, interactive=False, script=False
+    ):
+        """
+        @RETURN rc, out, err
+        """
+        if env or sudo or script:
+            raise j.exceptions.NotFound("Not implemented for ssh execuor")
+        return self.sshclient.execute(cmd, interactive, showout, replace, die, timeout)
 
-    # def executeRaw(self, cmds, die=True, showout=True, timeout=120):
-    #     rc, out, err = self.sshclient.execute(cmds, die=die, showout=showout, timeout=timeout)
-    #     return rc, out, err
+    @property
+    def uid(self):
+        return self.sshclient._id
```

Comment:
maybe better to use uid property github.com/threefoldtech/jumpscaleX_core/blob/a423e95f94/JumpscaleCore/clients/ssh/SSHClientBase.py#L86

Reply:
ok